pkg/eventcounter/api/api.go

```diff
@@ -151,16 +150,30 @@ func (s *eventCounterService) GetEvaluationCountBigQuery(
 		return nil, dt.Err()
 	}
 	variationCounts := s.convertEvaluationCounts(evaluationCounts, req.VariationIds)
-	s.logger.Debug("GetEvaluationCount result", zap.Any("rows", variationCounts))
-	return &ecproto.GetEvaluationCountV2Response{
-		Count: &ecproto.EvaluationCount{
-			FeatureId:      req.FeatureId,
-			FeatureVersion: req.FeatureVersion,
-			RealtimeCounts: variationCounts,
-		},
+	s.logger.Debug("GetExperimentEvaluationCount result", zap.Any("rows", variationCounts))
+	return &ecproto.GetExperimentEvaluationCountResponse{
+		FeatureId:       req.FeatureId,
+		FeatureVersion:  req.FeatureVersion,
+		VariationCounts: variationCounts,
 	}, nil
 }
 
+func validateGetExperimentEvaluationCountRequest(req *ecproto.GetExperimentEvaluationCountRequest) error {
+	if req.StartAt == 0 {
+		return localizedError(statusStartAtRequired, locale.JaJP)
```

Comment:
Please also use the `locale.NewLocalizer` for the error validation.

E.g.
https://github.com/bucketeer-io/bucketeer/blob/c5b6fedc867f9736b1eaaf4213573892b9377e06/pkg/account/api/validation.go#L162-L194